test/methods.jl

```diff
@@ -762,3 +762,10 @@ test = rebuild(ga; name = :test)
     @test_throws "strictly positive" Rasters.sample(StableRNG(123), test, 3, skipmissing = true, replace = false)
     @test_throws "Cannot draw" Rasters.sample(StableRNG(123), test, 5, replace = false)
 end
+
+@testset "extent" begin
+    ga = Raster(A, (X(1.0:1:2.0), Y(1.0:1:2.0)); missingval=missing) 
+    ext = extent(ga)
+    @test ext === Extent(X=(1.0,2.0), Y=(1.0,2.0))
+    @test extent(ext) === ext
```

Comment:
```suggestion
    @test Rasters._extent(ext) === ext
```
Shouldn't it be something like this?

Reply:
You're right, it should be.